lomap/graphgen.py

```diff
@@ -901,15 +963,23 @@ def max_dist_mol(mol):
             for each_node in self.resultGraph:
                 
                 id_mol = self.resultGraph.node[each_node]['ID']
-                mol = AllChem.RemoveHs(self.dbase[id_mol].getMolecule())
+                #sliu 07/17 skip remove Hs by rdkit if Hs cannot be removed
```

Comment:
Yes, I think that's the problem at RDKit end, I may give an example in the issue tracker to illustrate this issue.